test/full/fitcentroid/cylinder/cylinder.geo

```diff
@@ -0,0 +1,58 @@
+{
+name: "GEO",
+index: "world",
+valid_begin: [0, 0],
+valid_end: [0, 0],
+mother: "", // world volume has no mother
+type: "box",
+size: [20000.0, 20000.0, 20000.0], // mm, half-length
+material: "rock",
+invisible: 1,
+}
+
+{
+name: "GEO",
+index: "water",
```

Comment:
Maybe change index to match material? For both inner and outer water geometries.